Adyen/Utilities/KeyboardObserver.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// :nodoc:
+public protocol KeyboardObserver {
+
+    /// :nodoc:
+    func subscribeToKeyboardUpdates(_ observer: @escaping (_ keyboardRect: CGRect) -> Void) -> Any
+
+    /// :nodoc:
+    func removeObserver(_ observer: Any)
+}
+
+/// :nodoc:
+extension KeyboardObserver {
```

Comment:
Is this wise to hide `addObserver` behind protocol?
You need to delete observers on `deinit` and it is easy to forget if you didn't subscribed with your own hand..